src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.html

```diff
@@ -1,23 +1,23 @@
 <table
   *ngIf="customers"
-  class="table table-sm table-bordered table-striped mb-0"
+  class="table table-responsive-sm table-bordered table-striped mb-0"
```

Comment:
Responsive tables allow tables to be scrolled horizontally with ease. Make any table responsive across all viewports by wrapping a .table with .table-responsive. Or, pick a maximum breakpoint with which to have a responsive table up to by using .table-responsive{-sm|-md|-lg|-xl}.

example:

```
<div class="table-responsive">
  <table class="table">
    ...
  </table>
</div>
```

If you need use table-responsive-sm, the  correct form is; 

```
<div class="table-responsive-sm">
  <table class="table">
    ...
  </table>
</div>
```

[Documentation link](https://getbootstrap.com/docs/4.1/content/tables/)